android/src/ti/map/TiClusterMarker.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2013 by Appcelerator, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+
+package ti.map;
+
+import com.google.android.gms.maps.model.LatLng;
+import com.google.maps.android.clustering.ClusterItem;
+
+public class TiClusterMarker implements ClusterItem {
+
+    private LatLng position;
```

Comment:
Ideally indentation for the file would be with tabs to match the formatting we use for Titanium Java source.

Reply:
We could use clang-format here, yep! Simply a copy of the .clang-format file and let'd go. Probably after the initial review is approved, so the diff is easier to read.

Reply:
sorry, already did this after making the changes. Since it were mostly some left-overs and intent stuff